doc/ch-test_desc.rst

```diff
@@ -37,6 +37,10 @@ filesystem permissions fixtures must be created manually, in order to
 accommodate configurations where sudo is not available via the same login path
 used for running tests.
 
+The packed and unpacked image directories specified for testing are volatile.
+The contents of these directories are removed before each respective build and
+run phases.
+
```

Comment:
Suggest minor rephrase: "The packed and unpacked image directories are deleted before the build and run phases, respectively.".